vic/vic_run/src/runoff.c

```diff
@@ -48,8 +48,7 @@ runoff(cell_data_struct  *cell,
     size_t                     time_step;
     int                        last_index;
     int                        tmplayer;
-    size_t                     frost_area;
-    int                        fidx;
+    size_t                     fidx;
```

Comment:
See this related warning on the current build: https://travis-ci.org/UW-Hydro/VIC/jobs/103167674#L170-L174